clients/drcachesim/tracer/raw2trace.h

```diff
@@ -187,9 +182,8 @@ struct instr_summary_t final {
     byte length_ = 0;
     app_pc next_pc_ = 0;
 
-    std::vector<opnd_t> srcs_and_dests_;
-    uint8_t srcs_ = 0;
-    uint8_t dests_ = 0;
+    std::vector<opnd_t> mem_srcs_and_dests_;
```

Comment:
Add a comment explaining why this squishes the srcs and dests together.  (Otherwise someone is going to split them up, seeing no reason to combine them.)

Reply:
done